atd-vze/src/views/Crashes/CrashChangeLog.js

```diff
@@ -1,243 +1,161 @@
-import React, { Component } from "react";
-
+import React, { useMemo, useState } from "react";
 import {
-  Badge,
   Button,
   Card,
   CardBody,
   CardHeader,
-  Col,
   Table,
-  Row,
   Modal,
   ModalHeader,
   ModalBody,
   ModalFooter,
 } from "reactstrap";
 import { formatDateTimeString } from "../../helpers/format";
 
-class CrashChangeLog extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      modal: false,
-      modalBody: null,
-      dataFrom: null,
-      dataTo: null,
-      data: this.props.data,
-    };
-  }
-
-  /**
-   * Shows the modal box by changing the state, does not alter the body.
-   */
-  showModal = () => {
-    this.setState({
-      modal: true,
-    });
-  };
-
-  /**
-   * Closes the modal box by changing the state and emptying the modal body.
-   */
-  closeModal = () => {
-    this.setState({
-      modal: false,
-      modalBody: null,
-    });
-  };
-
-  /**
-   * Compares the archived json object against the current data object as populated via props.
-   * @param {Object} record's json
-   */
-  compare = record => {
-    // Holds only the keys
-    let diff = [];
-
-    // Holds the html for our modal box
-    let modalBody = null;
-
-    // Iterate through our record's key & values
-    for (let [key, value] of Object.entries(
-      this.props.data.atd_txdot_crashes[0]
-    )) {
-      // Let's get rid of typename
-      if (key === "__typename") continue;
-
-      try {
-        // Gather the archived value for the current field
-        let archivedRecordValue = record.record_json[key];
-
-        // If the value is different from the current value, then put it in the diff array.
-        if (archivedRecordValue !== value) {
-          diff.push({
-            original_record_key: key,
-            original_record_value: value,
-            archived_record_value: archivedRecordValue,
-          });
-        }
-      } catch (error) {
-        alert("Error: " + error);
-      }
+const KEYS_TO_IGNORE = ["updated_at", "updated_by", "position"];
```

Comment:
maybe in a future PR this can be moved to the `_column_metadata` table that we're aspiring to?